cli/internal/publish/images/images_test.go

```diff
@@ -366,3 +367,64 @@ func convertStringToRefKey(input map[string][]reference) map[listKey][]reference
 	}
 	return ret
 }
+
+func TestQuickContentType(t *testing.T) {
+	const dataDir = "testdata/content-type"
+
+	tests := []struct {
+		filename string
+		want     string
+		tryNoExt bool
+	}{
+		{
+			filename: "png.png",
+			want:     "image/png",
+			tryNoExt: true,
+		},
+		{
+			filename: "octetstream.svg",
+			want:     "image/svg+xml",
+		},
+		{
+			filename: "textplain.svg",
+			want:     "image/svg+xml",
+		},
```

Comment:
Is there value in checking (in case in a regression there's no `http.DetectContentType`) for the header logic? I don't mind of not.

The 🤖 says these are good for PNG (hex):

```
89 50 4E 47 0D 0A 1A 0A 00 00 00 0D 49 48 44 52
```

And this for JPG:

```
FF D8 FF E0 00 10 4A 46 49 46 00 01 01 01 00 48 00 48
```